### DISCUSSION TITLE
Volar instead of vetur

### DISCUSSION THREAD

**Discussion by LadIQe** [quasarframework/quasar#10532] [General - Components / Directives / etc]
Hi, did you try volar instead of vetur?

I have big problem with vetur. Typescript checking is not working correctly. Its incredibly slow, a lot of time I have to import things manually etc.

Do you have any recommendations for vetur settings or volar? Because volar is not working with quasar (at least component intellisense, so for example, it cant find q-dialog)

Thanks

**Comment by IlCallo** [comment]
We are working on Volar support, check out https://github.com/quasarframework/quasar/discussions/10619
